code/modules/unit_tests/species_whitelists.dm

```diff
@@ -1,5 +1,5 @@
 /datum/unit_test/species_whitelist_check/Run()
 	for(var/typepath in subtypesof(/datum/species))
 		var/datum/species/S = typepath
-		if(initial(S.changesource_flags) == NONE)
-			Fail("A species type was detected with no changesource flags: [S]")
+		if(initial(S.changesource_flags) == NONE && S.check_roundstart_eligible() == IMPOSSIBLE) //cannot be made in any way + cannot be roundstart = completely unavailable
+			Fail("A species type was detected with no changesource flags, and is not roundstart eligible: [S]")
```

Comment:
@optimumtact this alright? by default, check_roundstart_eligible will give IMPOSSIBLE if it's not enabled roundstart, but children can make the argument POSSIBLE which still passes the check, AKA it's not roundstart and cannot be obtained but it is enabled in an event, so it's OK.

Reply:
why are you doing it this way, just make another changesource flag that has no direct on station equivalent and call it EVENTRACE

Reply:
I like it, I'll do that

Reply:
this is trying to call a proc on a typepath